apps/client/src/routes/routes.tsx

```diff
@@ -0,0 +1,22 @@
+import DashboardList from '../pages/DashboardList';
+import Dashboard from '../pages/Dashboard';
+
+const routes = [
+  {
+    path: '/dashboards',
+    element: <DashboardList/>,
+    id: 'dashboard-list'
+  },
+  {
+    path: '/dashboards/:id',
+    element: <Dashboard/>,
+    id: 'dashboard-view'
+  },
+  {
+    path: '/dashboards/:id/edit',
+    element: <Dashboard/>,
+    id: 'dashboard-edit'
```

Comment:
Can we remove this edit route? It doesn't seem necessary unless we know we want a deep link.

Reply:
Agreed, we can hold off on this for now.

Reply:
Thank you!